CASL/Sublogic.hs

```diff
@@ -459,7 +459,7 @@ sl_form_level_aux ff (isCompound, leftImp) phi =
        let ql = sl_form_level_aux ff (isCompound, leftImp) f
        in if is_atomic_q q then ql else max FOL ql
    Junction j l _ -> maximum $ case j of
-      Con -> map (sl_form_level_aux ff (True, leftImp)) l
+      Con -> FOL : map (sl_form_level_aux ff (True, leftImp)) l
```

Comment:
This is problematic. The spec
```
spec sp =
  sort s
  preds p,r,q:s
  forall x:s . p(x) /\ q(x) => r(x)
end
```
is now classified as `Prenex`, but it should be `Horn`.